tsconfig.json

```diff
@@ -5,7 +5,6 @@
     "lib": ["dom", "es2020"],
     "declaration": true,
     "outDir": "dist",
-    "rootDir": "src",
```

Comment:
Was causing intellisense to shout when importing files from `src`. This doesn't affect building what we are outputting to `dist`